### PR TITLE
Modified eqemu_server.pl to use new login server schema

### PR DESCRIPTION
**Pull request by alimalkhalifa**
Has a matching pull request in Akkadius/EQEmuInstall

* Error was found by Maze

### CODE REVIEW

**Comment by Akkadius** [comment]
This needs to reference the source of truth here (Download the raw):

https://github.com/EQEmu/Server/blob/master/loginserver/login_util/login_schema.sql

**Comment by alimalkhalifa** [comment]
Updated to pull from source of truth

**Comment by alimalkhalifa** [comment]
Updated urls to use Variables

File: utils/scripts/eqemu_server.pl [hunk 1]
```diff
@@ -1518,9 +1518,9 @@ sub do_linux_login_server_setup {
     print "\n Done... \n";
 
     print "[Install] Pulling down Loginserver database tables...\n";
-    get_remote_file($install_repository_request_url . "login_server_tables.sql", "db_update/login_server_tables.sql");
+    get_remote_file("https://github.com/EQEmu/Server/blob/master/loginserver/login_util/login_schema.sql", "db_update/login_schema.sql");
```
Comment (Akkadius) [inline utils/scripts/eqemu_server.pl]: This needs to reference the raw file, not the Github HTML representation of this file

**Comment by alimalkhalifa** [comment]
Fixed, thanks for pointing out @Akkadius .  Also squashed the commits

**Pull request merged**